tests/e2e-autoscale/autoscale/cmd/verify/wait-and-validate-metrics.go

```diff
@@ -57,7 +53,17 @@ func main() {
 		os.Exit(1)
 	}
 
-	config, err := clientcmd.BuildConfigFromFlags("", kubeconfigPath)
+	kubeconfig := os.Getenv("KUBECONFIG")
+	if kubeconfig == "" {
+		kubeconfig = filepath.Join(homedir.HomeDir(), ".kube", "config")
+	}
```

Comment:
I would copy/paste the `getKubeconfigPath` function from the other file.

Reply:
I updated to use the same function in both the files. To get this case working in our CI jobs for OpenShift, we also need to change --duration in tests/e2e-autoscale/autoscale/03-install.yaml which I will do in our test executor script in CI and not in the upstream test code.